tasks/gen/action.ecr

```diff
@@ -0,0 +1,2 @@
+class <%= name %> < BrowserAction
+end
```

Comment:
I think it would be good to add the `action do/end` here. I think that's the most common thing to use, and it's easy to add `nested_` if you want a nested action instead.

```crystal
class <%= name %> < BrowserAction
  action do
  end
end
```

Reply:
Totally agree.  I misread the docs and thought you needed to explicitly enter the path in the action (e.g. `def "users/:user_id/announcements"`).  It's great that Lucky figures that out for you.

Reply:
Yep! As long as it is a resourceful action :) The generator could probably be smarter in the future and check for a resourceful action name, and if it isn't, then it would leave a comment in the template asking the user to declare their own custom route:

```crystal
class Users::Pricing < BrowserAction
  # This is not a RESTful action, so you'll need to declare your own route
  # https://luckyframework.org/guides/actions-and-routing/#routing
end
```

But that can come later